csfieldguide/static/interactives/image-bit-comparer/js/image-bit-comparer.js

```diff
@@ -14,7 +15,7 @@ ImageBitComparer.INITAL_IMAGES = [
                                   [image_bit_comparer_images["sunflower"], 'Sunflower'],
                                   [image_bit_comparer_images["temple-roof"], 'Roof of Temple'],
                                   [image_bit_comparer_images["nz-lake"], 'Lake in New Zealand'],
-                                  [image_bit_comparer_images["faces"], 'Faces'],
+                                  [image_bit_comparer_images["jacinda-ardern"], 'Jacinda Ardern'],
```

Comment:
Just to be consistent should we name the image 'woman' or something? The others are all unspecific